src/java.base/share/classes/java/lang/Long.java

```diff
@@ -42,6 +42,8 @@
 import static java.lang.String.LATIN1;
 import static java.lang.String.UTF16;
 
+import static jdk.internal.util.DecimalDigits.stringSize;
```

Comment:
Please don't use static import for methods. It makes it much harder to find where they come from.